binding.cc

```diff
@@ -893,17 +951,17 @@ NAPI_METHOD(db_open) {
   NAPI_ARGV_UTF8_NEW(location, 1);
 
   napi_value options = argv[2];
-  bool createIfMissing = BooleanProperty(env, options, "createIfMissing", true);
-  bool errorIfExists = BooleanProperty(env, options, "errorIfExists", false);
-  bool compression = BooleanProperty(env, options, "compression", true);
-
-  uint32_t cacheSize = Uint32Property(env, options, "cacheSize", 8 << 20);
-  uint32_t writeBufferSize = Uint32Property(env, options , "writeBufferSize" , 4 << 20);
-  uint32_t blockSize = Uint32Property(env, options, "blockSize", 4096);
-  uint32_t maxOpenFiles = Uint32Property(env, options, "maxOpenFiles", 1000);
-  uint32_t blockRestartInterval = Uint32Property(env, options,
+  const bool createIfMissing = BooleanProperty(env, options, "createIfMissing", true);
+  const bool errorIfExists = BooleanProperty(env, options, "errorIfExists", false);
+  const bool compression = BooleanProperty(env, options, "compression", true);
+
+  const uint32_t cacheSize = Uint32Property(env, options, "cacheSize", 8 << 20);
+  const uint32_t writeBufferSize = Uint32Property(env, options , "writeBufferSize" , 4 << 20);
+  const uint32_t blockSize = Uint32Property(env, options, "blockSize", 4096);
+  const uint32_t maxOpenFiles = Uint32Property(env, options, "maxOpenFiles", 1000);
+  const uint32_t blockRestartInterval = Uint32Property(env, options,
                                                  "blockRestartInterval", 16);
-  uint32_t maxFileSize = Uint32Property(env, options, "maxFileSize", 2 << 20);
+  const uint32_t maxFileSize = Uint32Property(env, options, "maxFileSize", 2 << 20);
```

Comment:
`const` :heart: Good thinking. Very good practice in c++ to use const as much as possible.